### PR TITLE
Perform a full rebuild of all sets

### LINKED ISSUE #378: Mudhole missing from card list
Mudhole appears to be missing from the list of cards. I have no idea why it's specifically this one. If someone knows where I can find a list of all cards that doesn't get its info from MTGJSON, I can check for others.

### PR DESCRIPTION
**Pull request by gwax**
~This PR depends on #376 and #377 for code fixes and should not be merged before those have been merged and this has been rebased.~

The major, substantive, change of this PR is the normalization of smart quotes to normal single and double quotes in card rulings and text.

Fix #378

### CODE REVIEW

**Comment by gwax** [comment]
This PR fixes #378 and a number of other problems with the current ODY build.

File: full_rebuild.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
+Howto: Perform a full rebuild
```
Comment (fenhl) [inline full_rebuild.rst]: Maybe this should be added to the [wiki](https://github.com/mtgjson/mtgjson/wiki) instead of the repo files?

Comment (fenhl) [inline full_rebuild.rst]: [Added.](https://github.com/mtgjson/mtgjson/wiki/How-to-perform-a-full-rebuild)

Comment (gwax) [inline full_rebuild.rst]: Cool, will remove from PR.

**Comment by gwax** [comment]
Hold off on merging, I'm going to clear my cache and build it all again to see if that will pick up the HOU masterpieces.

**Comment by tooomm** [comment]
>This PR fixes #378 and a number of other problems with the current ODY build.

I think you have to add the `fix(es) #378` tag to the starting post to make it automatically close issues on merge.

**Comment by silasary** [comment]
It must be in the commit message text, or in the body of the first post of a PR.

**Comment by fenhl** [comment]
What happened to this PR? Still rebuilding?

**Comment by gwax** [comment]
I've been having a dog of a time redownloading everything after clearing my cache. There also appears to be something broken in the processing of MPS_AKH.

I might end up fixing #313 and adding a retry before failing.

I hope to work on it some more this evening.

**Comment by ZeldaZach** [comment]
Any update on this?

**Comment by gwax** [comment]
Finally got it to build in full. This can be merged now.

File: web/changelog.json [hunk 2]
```diff
@@ -1,5 +1,15 @@
 [
   {
+    "version": "3.8.8",
```
Comment (ZeldaZach) [inline web/changelog.json]: Bump us to 3.9 please

**Pull request merged**